drivers/snmp-ups.c

```diff
@@ -392,6 +392,19 @@ void upsdrv_initups(void)
 		dstate_addcmd("shutdown.return");
 		dstate_addcmd("shutdown.stayoff");
 	}
+
+	/* Publish sysContact and sysLocation for all subdrivers */
+	/* sysContact.0 */
+	if (nut_snmp_get_str(".1.3.6.1.2.1.1.4.0", model, sizeof(model), NULL) == TRUE)
```

Comment:
Complementary thought:
while thinking back about that, and how to finalize implementation, I realized that:
1) these OIDs are read / write (some able to modify through upsrw, as long as you have write access to the device)
2) this implies that these OIDs must be resolvable (when issuing upsrw, the driver will resolve the NUT variable name into its OID), which implies that it has to be in MIB2NUT 
3) hence (2) implies that we should add it to all mapping tables
Or
Put in place a mechanism to add dynamically OIDs (or have some #include mechanism for DMFs).

Putting on hold for now